Framework/DataObjects/src/Peak.cpp

```diff
@@ -714,7 +714,9 @@ double Peak::getFinalEnergy() const { return m_finalEnergy; }
 double Peak::getInitialEnergy() const { return m_initialEnergy; }
 
 /** Get the difference between the initial and final neutron energy */
```

Comment:
In what units? I think there should be a comment about this. Can do as a follow up change.

Reply:
Nah might as well do it here. Initial and final are in meV so should also be in meV.